### PR TITLE
add lesson 9 solutions. completed!

### PR DESCRIPTION
**Pull request by sctcooper**

### CODE REVIEW

File: lesson-9-maximum-slice-problem/maxslicesum.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def solution(A):
+    max_ending = max_slice = -1000000
```
Comment (kiosklabs) [inline lesson-9-maximum-slice-problem/maxslicesum.py:2]: why is it set to this value?

File: lesson-9-maximum-slice-problem/maxslicesum.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    for a in A:
```
Comment (kiosklabs) [inline lesson-9-maximum-slice-problem/maxslicesum.py:3]: probably add more descriptive variable?

File: lesson-9-maximum-slice-problem/maxslicesum.py (hunk 1 above)
Comment (sctcooper) [inline lesson-9-maximum-slice-problem/maxslicesum.py:2]: that's the minimum range given in the problem. assigning it increases space consumption, and it's only an initial value

File: lesson-9-maximum-slice-problem/maxslicesum.py (hunk 2 above)
Comment (sctcooper) [inline lesson-9-maximum-slice-problem/maxslicesum.py:3]: well, idk, it seems appropriate though. seeing that a is an element of A... it doesn't get used much either way

**Pull request merged**